assets/javascript/giphy.js

```diff
@@ -0,0 +1,114 @@
+// giphy API key 6Ws2L6DO87tHU6XxSqZP6gyxoQaWgVn2
+var animals = [
+  "dog",
+  "cat",
+  "rabbit",
+  "hamster",
+  "skunk",
+  "goldfish",
+  "bird",
+  "ferret",
+  "turtle",
+  "sugar glider",
+  "chinchilla",
+  "hedgehog",
+  "hermit crab",
+  "gerbil",
+  "pygmy goat",
+  "chicken",
+  "capybara",
+  "teacup pig",
+  "serval",
+  "salamander",
+  "frog"
+];
+// Event listener for the GIF click events. Toggle its animation.
+function eventHandlerGifClicked() {
+  var state = $(this).attr("data-state");
+
+  if (state === "still") {
+    $(this).attr("src", $(this).attr("data-animate"));
+    $(this).attr("data-state", "animate");
+  } else if (state === "animate") {
+    $(this).attr("src", $(this).attr("data-still"));
+    $(this).attr("data-state", "still");
+  } else {
+    alert("Invalid State");
+  }
+}
+// Parse the results from the query and insert the GIFs
+function parseResponse(response) {
+  //   console.log(response);
+  var results = response.data;
+  //   console.log(results);
```

Comment:
If you've commented out the code, it's better to just delete it. For cleanliess!